multiapps-controller-process/src/main/java/org/cloudfoundry/multiapps/controller/process/steps/DeleteServiceStep.java

```diff
@@ -87,6 +102,18 @@ protected String getStepErrorMessageAdditionalDescription(ProcessContext context
         return ExceptionMessageTailMapper.map(configuration, CloudComponents.SERVICE_BROKERS, offering);
     }
 
+    private Metadata getMetadataWithoutMtaFields(Metadata metadata) {
```

Comment:
This method can be extracted in util class. It can be reused for different cf entities with metadata.